### PR TITLE
Switch

### PR DESCRIPTION
**Pull request by Kartynnikk**

### CODE REVIEW

File: switch.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Switch</title>
+</head>
+
+<body>
+
+    <script>
+        let browser;
+        if (browser == "Edge") {
```
Comment (kathrynowy) [inline switch.html:15]: strict comparison

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: switch.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                alert("Осень");
+                break;
+            default:
+                alert("Неправильное значение")
+        }
+    </script>
+
+</body>
+
+</html>
```
Comment (kathrynowy) [inline switch.html:68]: eof

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: switch.html (hunk 1 above)
Comment (Kartynnikk) [reply switch.html:15]: fixed

File: switch.html (hunk 2 above)
Comment (Kartynnikk) [reply switch.html:68]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**